featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -36,38 +38,37 @@ class DeepFeatureSynthesis(object):
 
                     [:class:`synthesis.dfs_filters.TraverseUp`]
 
-            agg_primitives (list[:class:`.primitives.AggregationPrimitive`], optional):
+            agg_primitives (list[str or :class:`.primitives.AggregationPrimitive`], optional):
                 list of Aggregation Feature types to apply.
 
-                Default:[:class:`Sum <.primitives.Sum>`, \
-                         :class:`Std <.primitives.Std>`, \
-                         :class:`Max <.primitives.Max>`, \
-                         :class:`Skew <.primitives.Skew>`, \
-                         :class:`Min <.primitives.Min>`, \
-                         :class:`Mean <.primitives.Mean>`, \
-                         :class:`Count <.primitives.Count>`, \
-                         :class:`PercentTrue <.primitives.PercentTrue>`, \
-                         :class:`NUniqe <.primitives.NUnique>`, \
-                         :class:`Mode <.primitives.Mode>`]
-
-            trans_primitives (list[:class:`.primitives.TransformPrimitive`], optional):
-                list of Transform Feature functions to apply.
-
-                    Default:[:class:`Day <.primitives.Day>`, \
-                             :class:`Year <.primitives.Year>`, \
-                             :class:`Month <.primitives.Month>`, \
-                            :class:`Weekday <.primitives.Weekday>`, \
-                             :class:`Haversine <.primitives.Haversine>`, \
-                             :class:`NumWords <.primitives.NumWords>`, \
-                             :class:`NumCharacters <.primitives.NumCharacters>`]
-
-            where_primitives (list[:class:`.primitives.AggregationPrimitive`], optional):
-                only add where clauses to these types of Aggregation
-                Features.
+                Default: ["sum", \
```

Comment:
Nitpick - I dont think these all need their own line. can we reform to be 80 chars?

Reply:
It's 114 chars (agg defaults) and 104 chars (trans defaults). Keep it on one line anyways? Even removing "    Default:" isn't enough to get it under 80.